src/Uno.UWP/ApplicationModel/Resources/ResourceLoader.cs

```diff
@@ -48,6 +48,23 @@ public ResourceLoader()
 
 		public string GetString(string resource)
 		{
+			
+			// "/[file]/[name]" format support
+			if(resource.ElementAt(0) == '/')
```

Comment:
This may fail if GetString is null or empty.
```suggestion
			if(resource.ElementAtOrDefault(0) == '/')
```